src/utils/internal/getCallFrame.ts

```diff
@@ -2,25 +2,22 @@
  * Return the stack trace frame of a function's invocation.
  */
 export function getCallFrame() {
-  try {
-    const inspectionError = new Error()
-    inspectionError.name = 'Inspection Error'
-    throw inspectionError
-  } catch (error) {
-    const frames: string[] = error.stack.split('\n')
+  const frames: string[] = (new Error().stack as string).split('\n')
 
-    // Get the first frame that doesn't reference the library's internal trace.
-    // Assume that frame is the invocation frame.
-    const declarationFrame = frames.slice(1).find((frame) => {
-      return !/(node_modules)?\/lib\/(umd|esm)\//.test(frame)
-    })
+  // Get the first frame that doesn't reference the library's internal trace.
+  // Assume that frame is the invocation frame.
+  const rePathIsNotUseful = /(node_modules)?[\/\\]lib[\/\\](umd|esm)[\/\\]|^[^\/\\]*$/
```

Comment:
So `node_modules` part is not necessary? Interesting. Could you share more info when does this happen?

Reply:
I didn't change anything about that part, `(node_modules)?` was there already in commit 944d805

Reply:
But thinking about it, it probably wouldn't make any difference if we remove that part from the regex, `/(node_modules)?[\/\\]lib[\/\\](umd|esm)[\/\\]|^[^\/\\]*$/` wouldn't match anything more than `/[\/\\]lib[\/\\](umd|esm)[\/\\]|^[^\/\\]*$/` would... I think 🤔

Reply:
Double but, what if someone will use `/lib/esm/msw.js` file structure to set up the handlers in their repo?  While wouldn't want to ignore that and in their case `msw` library would be inside node_modules indeed .. but I cannot think of a simple way how to make this testable inside this repo as well as handle such a special edge case.

One idea was to detect and ignore `msw` in the path, but there are both false positives (someone using "msw" as a directory name to set up msw handlers) and false negatives (someone cloning this repo into a folder different from "msw").

Unless someone has a better idea for more reliable detection of "internal msw" vs "user space", IMHO let's ignore that for now 😰